daemon/info_unix.go

```diff
@@ -173,8 +173,8 @@ func (daemon *Daemon) fillPlatformInfo(v *system.Info, sysInfo *sysinfo.SysInfo,
 	}
 }
 
-func (daemon *Daemon) fillPlatformVersion(v *types.Version, cfg *configStore) {
-	if rv, err := daemon.containerd.Version(context.Background()); err == nil {
+func (daemon *Daemon) fillPlatformVersion(ctx context.Context, v *types.Version, cfg *configStore) {
+	if rv, err := daemon.containerd.Version(ctx); err == nil {
```

Comment:
We should handle `err != nil` here and check if the context was cancelled.

Reply:
Also, this function still has other `log.G(context.TODO())`, which could use the passed ctx.